src/reducers/createStore.js

```diff
@@ -0,0 +1,5 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+export default function configureStoreApp(reducer) {
+  return configureStore(reducer);
```

Comment:
I think we don't need this, let's create the store in `src/index.js` as mentioned above.